recipes/pangolin/meta.yaml

```
@@ -39,6 +40,7 @@ test:
   commands:
     - pangolin -h
     - pangolin --all-versions
+    - TMPD=$(mktemp -d) && pangolin --update-data --datadir "$TMPD"
```

Can this be simplified to this?
```suggestion
    - pangolin --update-data --datadir "$(mktemp -d)"
```